### PR TITLE
ref Task #3685 [Movie Detail] [Movie Detail] Handle event on Movie Screen

### PR DESCRIPTION
**Pull request by junitvn**
https://edu-redmine.framgia.vn/issues/3675

### CODE REVIEW

**Comment by junitvn** [comment]
ready

**Comment by junitvn** [comment]
ready

**Comment by junitvn** [comment]
Ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,6 +22,7 @@
         <activity android:name=".screen.search.SearchActivity" />
         <activity android:name=".screen.more.MoreMovieActivity" />
         <activity android:name=".screen.movie_detail.MovieDetailActivity" />
+        <activity android:name=".screen.movie_detail.PlayTrailerActivity"></activity>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: self close tag />

Comment (DoanVanToan) [reply app/src/main/AndroidManifest.xml]: rename to TrailerActivity

File: app/src/main/java/com/framgia/moviedb/screen/movie_detail/MovieDetailPresenter.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        mView = view;
+        mRepository = repository;
+    }
+
+    @Override
+    public void getMoreInfo(String movieId) {
+        mRepository.getMovie(movieId, new MoviesDataSource.GetMovieCallback() {
+            @Override
+            public void onMovieLoaded(MovieDetail movieDetail) {
+                mView.showReleaseDate(movieDetail.getReleaseDate().substring(0, 4));
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/moviedb/screen/movie_detail/MovieDetailPresenter.java]: class này có nhiều hardcode

**Comment by DoanVanToan** [comment]
commented

**Comment by junitvn** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**